pkg/server/plugin/nodeattestor/x509pop/x509pop_test.go

```diff
@@ -100,231 +96,188 @@ func (s *Suite) TestAttestSuccess() {
 	for _, tt := range tests {
 		tt := tt // alias loop variable as it is used in the closure
 		s.T().Run(tt.desc, func(t *testing.T) {
-			log.Printf("Test %q - conf : %s", tt.desc, tt.giveConfig)
-			s.configure(tt.giveConfig)
+			attestor := s.loadPlugin(t, tt.giveConfig)
 
 			require := s.Require()
```

Comment:
Hmm, this is wrong. If there is a failure it will fail the parent testing.T. Mind fixing this while you are in there?
```suggestion
			require := require.New(t)
```